.github/workflows/proto-sync.yml

```diff
@@ -0,0 +1,72 @@
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'packages/core/protocols/src/proto/**'
+  workflow_dispatch:
+
+name: sync-proto
+
+jobs:
+  copy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Get current commit
+        run: echo "COMMIT_HASH=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
+
+      - name: Get new branch name
```

Comment:
would it be more convenient to have a well-known dxos-sync branch (like a feature brach) that we push to -- and pull on the other side as needed?

Reply:
i was considering a well-known branch as well, but eventually though it's better to have a new branch each time, since there is a chance that it will be a breaking change, and someone will have to manually update KUBE's code in order to actually make things work, so with the new-branch-each-time that person could be sure that if he stuck in the middle his changes won't get erased by the next upcoming proto update. In that case, he does not need to pull anything - just to take over that PR that action opens for him